### ISSUE TITLE
Implement new FP encoding proposal

### ISSUE THREAD

**Issue by ccelio** [riscv-boom/riscv-boom#24]
Proposal here: https://groups.google.com/a/groups.riscv.org/d/msg/isa-dev/_r7hBlzsEd8/cWPyJKMzCQAJ

Also see (https://github.com/ucb-bar/rocket-chip/pull/666).

**Summary:** close up undefined behavior in the ISA spec regarding FP loads/stores of recoded `n<FLEN` values.

**Comment by ccelio** [comment]
This has been addressed by https://github.com/ucb-bar/riscv-boom/commit/98c60153c6c49e15aeb5664d702755dd6295fc7d on the devel branch.  Once that is merged to master, we can close this.

**Comment by ccelio** [comment]
(https://github.com/ucb-bar/riscv-boom/commit/98c60153c6c49e15aeb5664d702755dd6295fc7d) has now been merged into master. Co-simulation/commit log enthusiasts may now rejoice.
